examples/Makefile.am

```diff
@@ -15,13 +15,10 @@ dist_examples_DATA =   \
 if HAVE_CUDA
 EXAMPLE_CUDA_LD_FLAGS = $(CUDA_LDFLAGS)
 EXAMPLE_CUDA_LIBS = $(CUDA_LIBS)
-# cuda.h couldn't be compiled with -pedantic flag
-EXAMPLE_CUDA_CFLAGS =
```

Comment:
Actually most of our code already compiles without `-pedantic`, it was only here. I removed it here as well, otherwise the `UCS_BITMAP_INIT` macro won't compile.

Reply:
not sure why it has to be changed in this PR?

Reply:
Because we added the `UCS_BITMAP_INIT` macro in this PR. If we don't remove `-pedantic` now, the code can't compile unless `UCS_BITMAP_INIT` is removed.

Reply:
where is bitmap.h included from?

Reply:
"hello world" examples use UCP, thus they must make use (indirectly) of `bitmap.h`, thus compile it.

Reply:
they should not include bitmap.h, since it should not included in ucp.h